pkg/networking/cilium/installation.go

```diff
@@ -12,9 +14,15 @@ type Installation struct {
 	ConfigMap *corev1.ConfigMap
 }
 
-// Installed determines if all Cilium components are present.
-// If the ConfigMap doesn't exist we still considered Cilium is installed.
-// The installation might not be complete but it can be functional.
+// Installed determines if all EKS-A Embedded Cilium components are present. It identifies
+// EKS-A Embedded Cilium by the image name. If the ConfigMap doesn't exist we still considered
+// Cilium is installed. The installation might not be complete but it can be functional.
 func (i Installation) Installed() bool {
-	return i.DaemonSet != nil && i.Operator != nil
+	var isEKSACilium bool
+	if i.DaemonSet != nil {
+		for _, c := range i.DaemonSet.Spec.Template.Spec.Containers {
+			isEKSACilium = isEKSACilium || strings.Contains(c.Image, "eksa")
```

Comment:
This is ok for now due to time constrains, but what do you think about replacing this by an annotation in the DS? It seems more robust that relying on a substring comparison.

I don't see why the image name wouldn't include `eksa` but we don't have control over that and it's subject to change and if it does, we would need to remember this is here.

Reply:
That sounds sensible. It shifts the concern toward deployment which seems saner than depending on an image build process to use a specific name.

I'll raise a ticket.